code/modules/tables/tables.dm

```diff
@@ -176,22 +176,16 @@
 
 /obj/structure/table/attack_generic(mob/user as mob, var/damage)
 	if(damage >= 10)
-		if(reinforced)
-			if(prob(50))
-				visible_message("<span class='danger'>\The [user] tears apart \the [src]!</span>")
-				src.break_to_parts()
-				user.do_attack_animation(src)
-				return 1
-			else
-				visible_message("<span class='danger'>\The [user] smashes against \the [src]!</span>")
-				take_damage(damage/2)
-				user.do_attack_animation(src)
-				..()
-				return 1
-		visible_message("<span class='danger'>\The [user] tears apart \the [src]!</span>")
-		src.break_to_parts()
-		user.do_attack_animation(src)
-		return 1
+		if(reinforced && prob(70))

```

Comment:
Just noting here, this is a 7 and not a 5, which makes it 20% less likely to shred it to bits. In case that wasn't intentional.

Reply:
That was intentional, as 50/50 makes it rather pointless for reinforcing.